src/docs/sdk/event-payloads/breadcrumbs.mdx

```diff
@@ -2,107 +2,146 @@
 title: Breadcrumbs Interface
 ---
 
-The Breadcrumbs Interface specifies a series of application events, or
-"breadcrumbs", that occurred before an event.
+Sentry uses _breadcrumbs_ to create a trail of events that happened prior to an issue. These events are very similar to traditional logs but can record more rich structured data.
 
-An [event](/sdk/event-payloads/) may contain one or more breadcrumbs in an attribute named `breadcrumbs`.
+This page provides technical information about a breadcrumb structure. You can read an overview of manual breadcrumb recording and customization on our [Breadcrumbs](https://docs.sentry.io/enriching-error-data/breadcrumbs/) sentry docs page.
 
-This interface is an object with a single `values` attribute containing an
-ordered list of breadcrumb objects. The entries are ordered from oldest to
-newest. Consequently, the last entry in the list should be the last entry before
-the event occurred.
+An event may contain one or more `breadcrumbs` properties. The entries are ordered from oldest to newest. Consequently, the last entry in the list should be the last entry before the event occurred.
 
-While breadcrumb attributes are not strictly enforced, a breadcrumb is most
-useful when it includes at least a `timestamp` and `type`, `category` or
-`message`. The rendering of breadcrumbs in Sentry depends on what is provided.
+The following example illustrates the breadcrumbs part of the event payload and omits other attributes for simplicity.
 
-## Attributes
+```json
+{
+  "breadcrumbs": {
+    "values": [
+      {
+        "timestamp": "2016-04-20T20:55:53.845Z",
+        "message": "Something happened",
+        "category": "log",
+        "data": {
+          "foo": "bar",
+          "blub": "blah"
+        }
+      },
+      {
+        "timestamp": "2016-04-20T20:55:53.847Z",
+        "type": "navigation",
+        "data": {
+          "from": "/login",
+          "to": "/dashboard"
+        }
+      }
+    ]
+  }
+}
+```
+
+A breadcrumb object contains the property `values`, which is an array of objects with the following properties:
 
-`timestamp`
+`type` (optional)
 
-: _Recommended_. A timestamp representing when the breadcrumb occurred. The
-  format is either a string as defined in [RFC
-  3339](https://tools.ietf.org/html/rfc3339) or a numeric (integer or float)
-  value representing the number of seconds that have elapsed since the [Unix
-  epoch](https://en.wikipedia.org/wiki/Unix_time).  
-  Breadcrumbs are most useful when they include a timestamp, as it creates a
-  timeline leading up to an event.
+The type of breadcrumb.
```

Comment:
```suggestion
: The type of breadcrumb. By default, all breadcrumbs are recorded as `default`, which makes them appear as a Debug entry, but Sentry provides other types that influence how the breadcrumbs are rendered. For more information, see the [description of recognized breadcrumb types](#breadcrumb-types).
```

Reply:
Note the tiny change here that renders the term and definition for easier reading. A good example of this is in src/docs/platform/javascript/common/config/basic.mdx

Tl;dr Use this format:
`term`

: definition

The term is then bold and in code format with the definition that follows slightly indented.